### ISSUE TITLE
Episerver AdminUI JSON import does not work if language not active

### ISSUE THREAD

**Issue by nzsb** [valdisiljuconoks/localization-provider-epi#30]
We have the setup that in development environment  we have only 2 languages activated in Episerver CMS, but in production 14. We need to synchronise the resources in both ways. If we export JSON on production and try to import in development, we get the following message and the import is not possible:
'Importing language `cs, da, fr, hr, hu, it, nl, pl, ro, sk, sl, tr` is not available in current EPiServer installation.'

It seems to be an extra check, I do not really understand the reason for this, but it makes the import impossible with JSON. We do not want to activate the same set of languages in the target environment? All the resources in all languages were imported in development environment with the migration tool, so they are there and there should be no reason not to be able to import then.

Using DbLocalizationProvider.AdminUI.EPiServer version 5.6.0.

**Comment by valdisiljuconoks** [comment]
hi,

there was a reasoning behind this once import was available also for editors. idea was that you wouldn't be able to modify resource translations for languages that are not enabled / or you don't have permissions to. now when import is available for admins only - this limitation does not make sense anymore.

will be fixed in upcoming v5.7

thx!

**Comment by nzsb** [comment]
hi,

Great, thank you!

**Comment by nzsb** [comment]
Hi,

I updated the packages in our project to version 5.7.0, now I get the following error message when I try to import JSON: 'Import failed! Reason: Object reference not set to an instance of an object.'. I also get this message if the enabled languages are the same between systems, and also version 5.6.0 (I did not tested this scenario earlier).
Should I create a new issue? Where can I get some logs which can help you, in our Episerver Log I could not find anything.

Thanks.
